webapp/src/features/adminEditRsu/AdminEditRsu.tsx

```diff
@@ -112,11 +109,11 @@ const AdminEditRsu = (props: AdminEditRsuProps) => {
     },
   })
 
-  const { rsuData } = props
+  const { rsuIp } = useParams<{ rsuIp: string }>()
 
   useEffect(() => {
-    dispatch(getRsuInfo(rsuData.ip))
-  }, [dispatch, rsuData.ip])
+    dispatch(getRsuInfo(rsuIp))
```

Comment:
With the urlRouting, this allows invalid rsuIp to be "edited". We'll want to add a check for invalid edit attempts, as well as trying to directly edit ones they don't have permissions for.

The backing admin-rsu function is set to False on organization_required which I believe would mean a user could bypass our security here and edit an RSU by IP that they shouldn't be able to

Reply:
Added a check!